k8s/solution/answers.md

```diff
@@ -0,0 +1,26 @@
+# Lesson 1
+
+> Do you see any difference when using `kubectl` to delete the pod? 
+
+* Using the "Shutdown" button the application terminates itself, putting the pod into `completed` state.
+  After this the app is restarted within the *same* pod.
+* Killing a pod with `kubectl delete pod` kills the application and removes the pod from the list while
+  spawning a new pod (= application instance).
```

Comment:
This was more meant as a personal observation question :) But maybe still nice to see the answers here. Maybe we can really have a answeres&troubleshooting document that is allowed to have spoilers.